src/styles/components/_button.scss

```diff
@@ -1,54 +1,44 @@
-.bsds-button
-{
+.bsds-button {
   @include button-default;
 
-  &-assertive
-  {
+  &-assertive {
     @include button-assertive;
   }
 
-  &-subtle
-  {
+  &-subtle {
     @include button-subtle;
   }
 
-  &-ghost
-  {
+  &-ghost {
     @include button-ghost;
   }
 
-  &-icon
-  {
+  &-icon {
     @include button-icon;
   }
 
-  &-nav-back
-  {
+  &-nav-back {
     $icon-size: 1em;
+
```

Comment:
do we want to disable the extra line here?

Reply:
I kind of like it, separates the variables from the lines actually changing styles. Especially since variables in block are pretty uncommon in our code base.